.github/workflows/reusable-e2e.yml

```diff
@@ -108,9 +135,9 @@ jobs:
         with:
           comment-id: ${{ steps.fc.outputs.comment-id }}
           issue-number: ${{ github.event.pull_request.number }}
-          body-path: bundle-size-pr-comment.md
+          body-path: ${{ env.E2E_APP_DIR }}/bundle-size-pr-comment.md
           edit-mode: replace
-  bundle-size-check:
+  bundle-size-store:
```

Comment:
more expressive than "check" (could be confused with actually checking something, whilst its storing size in check runs)